pom.xml

```diff
@@ -1054,7 +1054,7 @@
                                     <repositoryId>${repository.id}</repositoryId>
                                     <url>${repository.url}</url>
                                     <groupId>com.enactor.chromium</groupId>
-                                    <artifactId>mac-binaries</artifactId>
+                                    <artifactId>mac-binaries-${distribution.arch}</artifactId>
```

Comment:
As above you should be using the <classifier> tag for this, rather than putting it in the artifact id